net/http/test/CMakeLists.txt

```diff
@@ -4,8 +4,12 @@ target_link_libraries(client_perf PRIVATE photon_shared)
 add_executable(server_perf server_perf.cpp)
 target_link_libraries(server_perf PRIVATE photon_shared)
 
-add_executable(pure_libcurl pure_libcurl.cpp)
-target_link_libraries(pure_libcurl PRIVATE photon_shared)
+if (PHOTON_ENABLE_CURL)
+    target_compile_definitions(client_perf PRIVATE ENABLE_CURL)
+
+    add_executable(pure_libcurl pure_libcurl.cpp)
+    target_link_libraries(pure_libcurl PRIVATE photon_shared)
```

Comment:
`pure_libcurl` depends only libcurl & gflags, not related to anything in photon, why link photon here?

Reply:
delete it